javascript/extractor/src/com/semmle/jcorn/ESNextParser.java

```diff
@@ -240,6 +240,9 @@ protected Expression parseExprAtom(DestructuringErrors refDestructuringErrors) {
     if (this.type == TokenType._import) {
       Position startLoc = this.startLoc;
       this.next();
+      if (this.type == TokenType.dot) {
```

Comment:
You could use `this.eat(TokenType.dot)` here, then you don't need the `this.next()` on line 424 below.